domains/certificates/RegistryConnector/Worker/Program.cs

```diff
@@ -54,7 +54,7 @@
 builder.Services.AddScoped<IKeyGenerator, KeyGenerator>();
 
 builder.Services.AddHealthChecks()
-    .AddNpgSql(builder.Configuration.GetConnectionString("Postgres")!);
+    .AddNpgSql(sp => sp.GetRequiredService<IConfiguration>().GetConnectionString("Postgres")!);
```

Comment:
Why this change?

Reply:
After upgrading a healthcheck nuget package, the old way does not work anymore, causing tests to fail. Therefore the serviceprovider is now used to register the npgsql health endpoint.